core/scm.js

```diff
@@ -115,15 +118,22 @@ const SCHEMA_HOOK = Joi.object().keys({
         .optional()
         .label('PR original source'),
 
-    prRef: Joi.string()
-        .optional()
-        .label('PR reference of the repository'),
+    scmContext: Joi
+        .string().max(128)
+        .required()
```

Comment:
If this is required, can this change still be backwards-compatible?

Reply:
We weren't using any validation before for parseHook, so the entire `hook` schema was never used a whole, just parts of it.
I was thinking of just bumping major for scm-base and scm-github after we pull in/use the new validation, but this change (in the data-schema) shouldn't break anything.